### PR TITLE
Fix region tags

### PR DESCRIPTION
**Pull request by ace-n**

### CODE REVIEW

**Comment by ace-n** [comment]
@kurtisvg please merge if LGTM - thanks!

File: functions/helloworld/scala-helloworld/pom.xml [hunk 1]
```diff
@@ -43,8 +48,8 @@
       <groupId>org.scala-lang</groupId>
       <artifactId>scala-library</artifactId>
       <version>2.13.2</version>
-    </dependency>
-
+    </dependency>=
```
Comment (kurtisvg) [inline functions/helloworld/scala-helloworld/pom.xml:51]: I think this is your test failure
```suggestion
    </dependency>
```

**Comment by kurtisvg** [comment]
@ace-n LGTM once tests pass

File: functions/helloworld/scala-helloworld/pom.xml (hunk 1 above)
Comment (ace-n) [reply functions/helloworld/scala-helloworld/pom.xml:51]: Oops - thanks!

**Pull request merged**